maven/org.eclipse.emf.mwe2.parent/pom.xml

```diff
@@ -209,6 +201,15 @@
 						<excludeInnerJars>true</excludeInnerJars>
 					</configuration>
 				</plugin>
+				<plugin>
+					<groupId>org.eclipse.tycho</groupId>
+					<artifactId>tycho-packaging-plugin</artifactId>
+					<version>${tycho-version}</version>
+					<configuration>
+						<format>${tycho-qualifier}</format>
+						<strictVersions>true</strictVersions>
```

Comment:
this is head for readability only correct?

Reply:
I moved the plugin's configuration from the build section to pluginManagement to make the effective POM more readable. When the release profile is activated, only the pluginManagement changes. Otherwise you see both configurations.